src/app/qgsrastercalcdialog.h

```diff
@@ -42,6 +42,10 @@ class APP_EXPORT QgsRasterCalcDialog: public QDialog, private Ui::QgsRasterCalcD
     QString outputFormat() const;
     QgsCoordinateReferenceSystem outputCrs() const;
     bool addLayerToProject() const;
+    //true if Virtual is checked
+    bool useVirtualProvider() const;
+    //return the name written in the qlineedit
+    QString virtualLayerName() const;
```

Comment:
When adding docs for functions, let's use `//!` or `/**  .... */` so that it is understood by Doxygen